.travis.yml

```diff
@@ -3,37 +3,6 @@ node_js:
 - '6.10'
 - '7'
 - '8'
-cache:
-  yarn: true
```

Comment:
From experimentation the other day, I found the fastest runs where where the `node_modules` directories were not cached themselves, but the `~/.cache/yarn` directory (which is set via the `yarn: true` alias) still was.

As such, I'd leave caching enabled, but use this config (uses the shorthand for when there is only one entry):
```yaml
cache: yarn
```
(From https://docs.travis-ci.com/user/caching/#yarn-cache)

I believe this works the best since the cache keeps the files in archives rather than extracted, and is also reused multiple times over (compared to the individual `node_modules` directories where each is only used once per package, and is fully extracted) - so gives the best bang for buck per MB/file count.